common/src/main/java/com/wynntils/models/containers/LootChestModel.java

```diff
@@ -102,18 +109,42 @@ public void onSetSlot(ContainerSetSlotEvent.Post event) {
                                 openedChestCount.get(),
                                 dryCount.get(),
                                 dryBoxes.get(),
+                                dryEmeralds.get(),
+                                dryItemTiers.get(),
                                 System.currentTimeMillis(),
                                 new Location(lastChestPos)));
                 mythicFinds.touched();
 
                 dryBoxes.store(0);
                 dryCount.store(0);
+                dryEmeralds.store(0);
+                dryItemTiers.store(new EnumMap<>(GearTier.class));
             }
         } else {
             dryBoxes.store(dryBoxes.get() + 1);
```

Comment:
If you moved the call to     progressItemFound() to here, you would not need to filter out mythics from there.

Reply:
Ah, right, I see now that it is not that simple...